demos/storybook/stories/drawer/with-nav-rail.stories.ts

```diff
@@ -5,19 +5,19 @@ const headerImg = require('../../assets/eaton-condensed.png');
 
 export const navItems: DrawerNavItem[] = [
```

Comment:
I'm not sure how you guys feel about this, but should we update these items/icons in the Drawer to be things that you might actually see in a drawer? E.g.:
- Overview
- Alarms
- Settings
- Device Settings

Reply:
How's this?

![image](https://user-images.githubusercontent.com/6538289/94019181-78f1ae00-fd7f-11ea-9a01-93f5ccf69d3f.png)

This is using the 16px padding as well.

Reply:
@huayunh @joebochill Is there a EATON svg for the DrawerHeader?  The image I'm using looks a bit blurry.

Reply:
Have to publish @pxblue/icons first for that eaton thing

The eaton logo is weird to be on the top, bc usually you'd expect those in the footer. Maybe use a pxblue logo instead and make it look more like a header (like using a blue background)?

Reply:
Kyle might have made some recommendations on what to use for those menu items in his storybook audit. I couldn't locate the miroboard though....

Reply:
I'm really glad you mentioned this @huayunh.  I moved the header content to the footer and found a bug.  The footer always thinks the drawer is closed and sets the visibility of the rail footer to hidden.  I'm pretty sure this issue applies to the subheader as well.  

Since some team(s?) have expressed interest in having conditional logic in the footer based on whether the drawer is opened/closed, does it make sense to remove the visibility logic from our drawer?

A user can pretty easily hide content themselves by just putting `*ngIf="open"` on the `<pxb-footer>`.  Thoughts @joebochill  @huayunh ??

Reply:
My 2c I'm thinking to remove the conditional visibility from our drawer.   I can do this refactor as part of this PR if this makes sense.

Reply:
I would not take the logic out until we have implemented a way for them to opt-in/opt-out. Otherwise it will break for people who are currently relying on it and we don't have a way for them to re-enable it (without doing it themselves).

Reply:
This could be a possible item to consider in the future; it'd have to be for all the Drawer implementations as well.  We can revisit this if teams are finding this feature inconvenient.